gaphor/core/styling/declarations.py

```diff
@@ -56,6 +56,12 @@ class Var(NamedTuple):
     name: str
 
 
+class PrefersColorScheme(Enum):
```

Comment:
This is a nice improvement to replace `dark_mode` :+1:

Reply:
It does make it easier to understand, doesn't it :).